Documentation/ceph-object.md

```diff
@@ -174,9 +174,9 @@ The following commands extract key pieces of information from the secret and con
 
 ```bash
 #config-map, secret, OBC will part of default if no specific name space mentioned
-export AWS_HOST=$(kubectl -n default get cm ceph-bucket -o yaml | grep BUCKET_HOST | awk '{print $2}')
-export AWS_ACCESS_KEY_ID=$(kubectl -n default get secret ceph-bucket -o yaml | grep AWS_ACCESS_KEY_ID | awk '{print $2}' | base64 --decode)
-export AWS_SECRET_ACCESS_KEY=$(kubectl -n default get secret ceph-bucket -o yaml | grep AWS_SECRET_ACCESS_KEY | awk '{print $2}' | base64 --decode)
+export AWS_HOST=$(kubectl -n default get cm ceph-bucket -o yaml | grep "^\s*BUCKET_HOST" | awk '{print $2}')
+export AWS_ACCESS_KEY_ID=$(kubectl -n default get secret ceph-bucket -o yaml | grep "^\s*AWS_ACCESS_KEY_ID" | awk '{print $2}' | base64 --decode)
+export AWS_SECRET_ACCESS_KEY=$(kubectl -n default get secret ceph-bucket -o yaml | grep "^\s*AWS_SECRET_ACCESS_KEY" | awk '{print $2}' | base64 --decode)
```

Comment:
What about using jsonpath here to access the keys?

For ConfigMap `kubectl -n default get cm ceph-bucket -o jsonpath='{.data.BUCKET_HOST}'.

For Secrets `kubectl -n default get secret ceph-bucket -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 --decode

Reply:
That's way better. I'll replace it with your solution.